app/controllers/schools/advice/boiler_control_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Schools
+  module Advice
+    class BoilerControlController < AdviceController
+      include AdvicePages
+
+      def show
```

Comment:
Am wondering if this redirect ought to be done once in the superclass rather than each controller - assuming it's a standard pattern? You have the advice_page_key which could be used in the redirect?